### PR TITLE
Add PHP Core Roundup #7

### PR DESCRIPTION
**Pull request by Ayesh**

### CODE REVIEW

File: source/_posts/2022-11-05-php-core-roundup-7.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+Additionally, Tim Düsterhus’s RFC to improve error handling semantics of the `unserialize` function recently finished the vote.
+
+There are quite a few more RFCs under discussion that target PHP 8.3, so it’s safe to say that the PHP 8.3 development is quite active even before we have the first GA PHP 8.2 release!
+
+
+## PHP 8.2 GA to be released this month!
+
+The first GA PHP version, PHP 8.2.0 is scheduled to be released on 24th of November. PHP 8.2 RC5 is already released, and RC6 (the last one) is scheduled for 10th of November.
+
+PHP 8.2 RC versions are available in Ondrej Sury’s Debian/Ubuntu repos, Remi’s repos for Fedora/RHEL, Docker images on Docker Hub, and compiled Windows binaries on windows.php.net.
```
Comment (saundefined) [inline source/_posts/2022-11-05-php-core-roundup-7.md:39]: Maybe add the link?

```suggestion
PHP 8.2 RC versions are available in Ondrej Sury’s Debian/Ubuntu repos, Remi’s repos for Fedora/RHEL, Docker images on Docker Hub, and compiled Windows binaries on [windows.php.net](https://windows.php.net). 
```

File: source/_posts/2022-11-05-php-core-roundup-7.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+
+## All PHP 7 Versions Reach EOL Next Month
+
+PHP 7’s journey comes to an end when PHP 8.2 is released. PHP 7.0 through 7.3 are no longer maintained and no longer receive security updates, but PHP 7.4 is currently receiving security updates from the PHP core developers. 
+
+
+## Recent RFCs, Merged PRs, Discussions, and Commits
+
+Changes and improvements to PHP are discussed, reported, and voted on by the PHP Foundation Team, the PHP development team, and contributors. Bug reports are made to the PHP [issue tracker](https://github.com/php/php-src/issues), changes are discussed in [mailing lists](https://www.php.net/mailing-lists.php), minor code changes are proposed as [pull requests](https://github.com/php/php-src/issues), and major changes are discussed in detail and voted on as [PHP RFCs](https://wiki.php.net/rfc). [Documentation](https://github.com/php/doc-en/) and the [php.net web site](github.com/php/web-php) changes are also discussed and improved at their relevant Git repositories on GitHub.
```
Comment (saundefined) [inline source/_posts/2022-11-05-php-core-roundup-7.md:49]:
```suggestion
Changes and improvements to PHP are discussed, reported, and voted on by the PHP Foundation Team, the PHP development team, and contributors. Bug reports are made to the PHP [issue tracker](https://github.com/php/php-src/issues), changes are discussed in [mailing lists](https://www.php.net/mailing-lists.php), minor code changes are proposed as [pull requests](https://github.com/php/php-src/issues), and major changes are discussed in detail and voted on as [PHP RFCs](https://wiki.php.net/rfc). [Documentation](https://github.com/php/doc-en/) and the [php.net web site](https://github.com/php/web-php) changes are also discussed and improved at their relevant Git repositories on GitHub.
```

**Review by saundefined — COMMENTED** [review]
Looks good 💙🐘

File: source/_posts/2022-11-05-php-core-roundup-7.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+	Bob Weinand created a new RFC and a [discussion](https://externals.io/message/118829) on allowing `[$a ?? $b] = $array;`, where `$a` is assigned `$b` if the key in the array is null or missing:
+	
+	```php
+	[$a ?? $b] = $array;
+	[$a ?? "default value"] = $array;
+	```
+	<br />
+	Currently, the snippets above result in Parse Error. The RFC proposes adding support for coalesce in array destructing (`list()` function calls, or `[$a, $b, …] = $array` syntax).
+
+	When destructing an array, the `$a` will be assigned ``$b```/"default value"` if `$array[0]` is not set.
```
Comment (TimWolla) [inline source/_posts/2022-11-05-php-core-roundup-7.md:85]: There a rendering error in this line.

File: source/_posts/2022-11-05-php-core-roundup-7.md [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
```
Comment (saundefined) [inline source/_posts/2022-11-05-php-core-roundup-7.md:85]:
```suggestion
	When destructing an array, the `$a` will be assigned ``$b``/``"default value"`` if `$array[0]` is not set.
```

**Pull request merged**